contentgrid-spring-boot-autoconfigure/src/main/java/com/contentgrid/spring/boot/autoconfigure/actuator/ActuatorAutoConfiguration.java

```diff
@@ -28,6 +29,7 @@
 
 @AutoConfiguration
 @ConditionalOnClass({PolicyActuator.class, WebHooksConfigActuator.class})
```

Comment:
```suggestion
@ConditionalOnClass({PolicyActuator.class, WebHooksConfigActuator.class, ConditionalOnAvailableEndpoint.class})
```

And here, we're making sure this thing is not imported when spring-boot-actuator-autoconfigure is not on the classpath